### ISSUE TITLE
Support for mixed SASS and SCSS projects

### ISSUE THREAD

**Issue by bcherny** [dlmanning/gulp-sass#199]
My project uses SASS, but consumes bourbon, which uses SCSS. Since I have to set the `indentedSyntax` flag for the full build, it seems that I can't currently use both.

**Comment by slavomirvojacek** [comment]
@bcherny I have encountered the same problem. As much as I hate to admit it, I have had to deal with it impromptu and implemented a hack. But it's a neat hack :)

```javascript
var	gulp = require("gulp"),
	sass = require("gulp-sass");

var sassSource = "css/source";
var options = {
	sass: { style: "nested", indentedSyntax: true },
	scss: { style: "nested" }
};

function compileSass (extension) {
	gulp.src(sassSource + "/include.sass") // This can also be include.scss
		.pipe(sass(options[extension]))
		.pipe(gulp.dest("css"));
}

gulp.task("sass", function () { compileSass("sass"); });
gulp.task("scss", function () { compileSass("scss"); });

gulp.task("default", function () {
	gulp.watch(sassSource + "/**/*.sass", ["sass"]);
	gulp.watch(sassSource + "/**/*.scss", ["scss"]);
});
```

**Comment by lmartins** [comment]
While `sass` syntax is supposed to have feature parity with `scss` my feeling is that edge cases will be way more frequent in `sass` syntax. That being said, I believe this is related to the compiler itself (libsass) and not with `gulp-sass`.
Maybe @dlmanning can confirm this.

**Comment by Keats** [comment]
We could use the file extension to pass https://github.com/sass/node-sass#indentedsyntax on a per-file basis I suppose

**Comment by Keats** [comment]
It seems like this is solved directly by node-sass/libsass in the next version, can you (@bcherny or @slavomirvojacek ) try the 2.x branch of gulp-sass and report on #222  of any bugs happening relating to that?

**Comment by Snugug** [comment]
Resolved by #222
